app/models/user.rb

```diff
@@ -71,6 +88,14 @@ def try_to_create_potential_swap
     return potential_swaps.create(target_user: target_user)
   end
 
+  def marginal_polls
+    Poll.where(["marginal_score < ?", 1000]).where(party: preferred_party).order(:marginal_score)
```

Comment:
Aren't we randomly sampling from this list?  In which case the ordering is pointless?  Or am I missing something?

Reply:
Yes ... in earlier stages of hacking I had an idea about weighting the randomisation towards low marginal scores, rather than applying a cutoff ... plus it was useful in the console. now it should go.

Reply:
Change just pushed